README.md

```diff
@@ -27,15 +27,14 @@ For local development:
 ```
 npm start
 ```
+This will compile the site into the \_sites directory and run a local file server where you can preview the site: http://localhost:8080
```

Comment:
Typo: It appears to compile to `_site`. Also not sure why there's a backslash there.

Reply:
The `s` was a typo (fixed). The `\` is to escape the underscore (so it doesn't italicize the rest of the sentence)